microcloud/test/includes/microcloud.sh

```diff
@@ -498,6 +498,10 @@ reset_system() {
 
     lxc exec "${name}" -- ip link del lxdfan0 || true
 
+    # Resync the time in case it got out of sync with the other VMs.
```

Comment:
This seem suspect. Any ideas @simondeziel ?

Reply:
@masnax is this due to VM snapshots being restored or something similar?

Reply:
Actually no. It seems they drift while staying up too. I occasionally see a 5 minute drift on the certs that are generated when the state directory doesn't exist.

Reply:
It's just hovering at 0.0-0.1, occasionally hitting 0.3. But you may have a point, it was particularly noticeable when I was debugging dqlite by having 10 nodes concurrently join the cluster over and over and over.